components/shared/workCard/WorkCard.tsx

```diff
@@ -99,18 +109,15 @@ const WorkCard = ({ work, className, isWithTilt = false }: WorkCardProps) => {
   const isSomeMaterialTypePodcast = work.materialTypes.some(materialType => {
     return materialType?.materialTypeGeneral?.code === "PODCASTS"
   })
-  const isSomeManifestationTypeCostFree = filteredManifestations.some(
+
+  const isSomeManifestationTypeCostFree = filteredManifestationsByAccessType.some(
     manifestation => manifestation.publizonData?.costFree
   )
   // sort manifestations by materialTypeSortPriority
-  const sortedManifestations = filteredManifestations.sort((a, b) => {
+  const sortedManifestations = filteredManifestationsByAccessType.sort((a, b) => {
```

Comment:
I think this sorting should also be somewhere globally accessible from a helper. We will need it for example on the search results to find out where to redirect to, right?